src/Uno.UI.Tests/Windows_Globalization/Given_NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,1570 @@
+			yield return new object[] { "vun", "Latn", "vun" };
+			yield return new object[] { "vun-TZ", "Latn", "vun-Latn-TZ" };
+			yield return new object[] { "wae", "Latn", "wae" };
+			yield return new object[] { "wae-CH", "Latn", "wae-Latn-CH" };
+			yield return new object[] { "cy", "Latn", "cy" };
+			yield return new object[] { "cy-GB", "Latn", "cy" };
+			yield return new object[] { "wal", "Latn", "wal" };
+			yield return new object[] { "wal-ET", "Latn", "wal-Ethi-ET" };
+			yield return new object[] { "wo", "Latn", "wo-Latn" };
+			yield return new object[] { "wo-SN", "Latn", "wo-Latn" };
+			yield return new object[] { "xh", "Latn", "xh" };
+			yield return new object[] { "xh-ZA", "Latn", "xh" };
+			yield return new object[] { "yav", "Latn", "yav" };
+			yield return new object[] { "yav-CM", "Latn", "yav-Latn-CM" };
+			yield return new object[] { "ii", "Latn", "ii" };
+			yield return new object[] { "ii-CN", "Latn", "ii-Yiii" };
+			yield return new object[] { "yi-001", "Latn", "yi" };
+			yield return new object[] { "yo", "Latn", "yo" };
+			yield return new object[] { "yo-BJ", "Latn", "yo-BJ" };
+			yield return new object[] { "yo-NG", "Latn", "yo-Latn" };
+			yield return new object[] { "dje", "Latn", "dje" };
+			yield return new object[] { "dje-NE", "Latn", "dje-Latn-NE" };
+			yield return new object[] { "zu", "Latn", "zu" };
+			yield return new object[] { "zu-ZA", "Latn", "zu" };
+		}
+
+		[DataTestMethod]
+		[DataRow("")]
+		[DataRow("abcd")]
+		[DataRow("Arab ")]
+		public void When_NumeralSystemIsInvalid_Then_Throw(string numeralSystem)
+		{
+			NumeralSystemTranslator numeralSystemTranslator = new NumeralSystemTranslator();
+			Assert.ThrowsException<ArgumentException>(() => numeralSystemTranslator.NumeralSystem = numeralSystem);
+		}
+
+		[TestMethod]
+		public void When_NumeralSystemIsNull_Then_Throw()
+		{
+			Assert.ThrowsException<ArgumentNullException>(() => new NumeralSystemTranslator { NumeralSystem = null });
+		}
+
+
+		[DataTestMethod]
+		[DataRow(new string[0])]
+		[DataRow(new string[] { "abcd" })]
+		[DataRow(new string[] { "en-US", "abcd" })]
+		public void When_LanguagesIsInvalid_Then_Throw(IEnumerable<string> languages)
+		{
+			Assert.ThrowsException<ArgumentException>(() => new NumeralSystemTranslator(languages));
+		}
+
+		[TestMethod]
+		public void When_LanguagesIsNull_Then_Throw()
+		{
+			Assert.ThrowsException<NullReferenceException>(() => new NumeralSystemTranslator(null));
+		}
+
+		[DataTestMethod]
+		[DynamicData(nameof(When_UsingLanguage_Then_InitializeCorrectly_TestData), DynamicDataSourceType.Method)]
+		public void When_UsingLanguage_Then_InitializeCorrectly(string languageTag, string numeralSystem, string resolveLanguage)
+		{
+			var numeralSystemTranslator = new NumeralSystemTranslator(new string[] { languageTag });
+			Assert.AreEqual(numeralSystem, numeralSystemTranslator.NumeralSystem);
+			Assert.AreEqual(resolveLanguage, numeralSystemTranslator.ResolvedLanguage);
+		}
```

Comment:
@jeromelaban I figure out that Android UI Tests and iOS Automated Runtime Tests failed because of the error below:
Failed: When_UsingLanguage_Then_InitializeCorrectly() [System.Reflection.TargetParameterCountException] 
System.Reflection.TargetParameterCountException: Number of parameters specified does not match the expected number.

but this test passed perfectly when target framework is NET461 or UWP. I don't know what is the reason it failed on Android and IOS. Can you help me?

Reply:
It fails on iOS/Android because `DynamicData` is not yet supported by the unit test engine there. `DataRow` is supported, though:

https://github.com/unoplatform/uno/blob/b18a4d0355a07731a3e6d88f058c0db69bfe7567/src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs#L566